deptry/cli.py

```diff
@@ -212,6 +220,17 @@ def deptry(
         logging.warning("Missing argument ROOT. E.g. `deptry .`")
         sys.exit(1)
 
+    virtual_env = os.environ.get("VIRTUAL_ENV")
+    logging.warning(root.absolute().name)
+    if not in_project_virtualenv(
+        root.absolute().name, prefix=sys.prefix, base_prefix=sys.base_prefix, active_virtual_env=virtual_env
```

Comment:
There are quite some arguments here, not all of them are clear to me. what are prefix and base_prefix? Do they really need to be arguments, or cshould we hardcode them in the method body?

Reply:
See my comment below 🙂